.eslintrc

```diff
@@ -0,0 +1,35 @@
+env:
+  node: true
+  browser: true
+
+rules:
+  brace-style: 2
+  comma-spacing: 1
+  eol-last: 0
+  indent: [2, 2]
+  key-spacing: 0
+  new-parens: 2
+  no-debugger: 2
+  no-dupe-args: 2
+  no-dupe-keys: 2
+  no-duplicate-case: 2
+  no-ex-assign: 2
+  no-extra-parens: 1
+  no-fallthrough: 2
+  no-multi-spaces: [2, { exceptions: { "ObjectExpression": true } }]
+  no-reserved-keys: 2
+  no-unreachable: 2
+  padded-blocks: [2, "never"]
+  quotes: [2, "single", "avoid-escape"]
+  semi: 2
+  space-after-keywords: 2
+  space-before-blocks: 2
+  space-before-function-paren: [2, "never"]
```

Comment:
```js
function name(args) {}
// instead of
function name (args){}
```
is used by node and also [suggested](http://javascript.crockford.com/code.html) by Crockford (js inventor):
> There should be **no space** between the name of a function and the  `(`
> There should be **one space** between the `)` and the `{`